cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -28,6 +28,76 @@ import (
 	"github.com/operator-framework/operator-sdk/internal/generate/collector"
 )
 
+const (
+	//nolint:lll
+	examples = `
+  # Using the example 'memcached-operator' and assuming a directory structure
+  # similar to the following exists:
+  $ tree api/ config/
+  api/
+  └── v1alpha1
+      ├── groupversion_info.go
+      ├── memcached_types.go
+      └── zz_generated.deepcopy.go
+  config/
+  ├── bundle
+  │   └── kustomization.yaml
+  ├── crd
+  │   ├── bases
+  │   │   └── cache.my.domain_memcacheds.yaml
+  │   ├── kustomization.yaml
+  │   ├── kustomizeconfig.yaml
+  │   ...
+  ├── default
+  │   ├── kustomization.yaml
+  │   ...
+  ├── manager
+  │   ├── kustomization.yaml
+  │   └── manager.yaml
+  ...
+
+  # Generate bundle files and build your bundle image with this 'make' recipe:
+  $ export USERNAME=<your registry username>
+  $ make bundle-build BUNDLE_IMG=quay.io/$USERNAME/memcached-operator-bundle:v0.0.1
```

Comment:
We have now 2 makefile target which make confuse. See:

```
# Build the bundle OCI image
bundle-build: manifests
	$(KUSTOMIZE) build config/bundle | operator-sdk generate bundle -q --manifests --metadata --overwrite --version $(VERSION)
	operator-sdk bundle validate config/bundle
	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .

# Generate a bundle directory
bundle: manifests
	$(KUSTOMIZE) build config/bundle | operator-sdk generate bundle -q --manifests --version $(VERSION)
```

Could we not remove the make bundle and just keep the make build-bundle instead of it which has been used in the example? Why keep both?

Reply:
Having one to scaffold a bundle and another to build a bundle image seems straightforward no?